ecms_base/ecms_base.profile

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Provides update hooks for previously installed sites.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Update Basic HTML configuration to match the install config.
+ */
+function ecms_base_update_9001(array &$sandbox): void {
```

Comment:
Nice! 

One question, does this also run in the `ecms_acquia` profile? If not, we may want to create a new `ecms_distribution` module and put these updates in a service class that then gets called from both profiles.

I have a feeling we'll be using that service class a lot.

Reply:
Ok I spun up a new site on DEV, then deployed a test branch of the distribution with this branch specified. Looks to have run! http://updatetest.dev-riecms.acsitefactory.com/admin/config/content/formats/manage/basic_html